test/egld-esdt-swap/mandos/wrap_egld.scen.json

```diff
@@ -39,6 +39,14 @@
                 "status": "0",
                 "message": "",
                 "logs": [
+                    {
+                        "address": "sc:egld-esdt-swap",
+                        "identifier": "str:ESDTTransfer",
+                        "topics": [
+                            "str:WEGLD-abcdef", "500",  "address:user"
```

Comment:
Format list as one element per line, for readability and consistency with the other Mandos tests.

Reply:
done.